spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -325,8 +328,6 @@ public void testTrustedPackages() throws Exception {
 				producerBindingProperties);
 
 		ExtendedConsumerProperties<KafkaConsumerProperties> consumerProperties = createConsumerProperties();
-		consumerProperties.getExtension()
-				.setTrustedPackages(new String[] { "org.springframework.util" });
```

Comment:
Why do we no longer need to test this property?

Reply:
Actually, we were not doing any assertions on that. I will add it back and assert.

Reply:
Sorry, my mistake. We were doing assertions on the mimetype from that package. Will restore.